utils/chroma_store.py

```diff
@@ -125,27 +132,27 @@ def list_collection_names(self) -> List[str]:
             return []
         return [collection.name for collection in collections]
 
-    def query_collection(
+    def query_collection(  # type: ignore
```

Comment:
A better way to handle this would be, adding `Any` to `kwargs` i.e. `**kwargs: Any`.